xivo_dao/alchemy/endpoint_sip.py

```diff
@@ -335,3 +335,18 @@ def _find_first_value(self, section, key):
         matching_options = section.find(key)
         for _, value in matching_options:
             return value
+
+    @hybrid_property
+    def webrtc(self):
+        # NOTE(fblackburn): It doesn't support many webrtc on the same endpoint
+        for key, value in self.endpoint_section_options:
+            if key == 'webrtc' and value == 'yes':
+                return True
+            if key == 'webrtc' and value == 'no':
+                return False
+
+        for template in self.templates:
```

Comment:
I think there a bug here.
Given an endpoint without webrtc.
Given a template disable webrtc
Given a template enable webrtc

the endpoint has no webrtc, then it goes through templates. The first one has webrtc = `False` which is not `True` but the other templates should not be looked at since `webrtc` has been defined.

Reply:
Yes it doesn't take template priority into account. It's necessary?